airgun/views/common.py

```diff
@@ -38,6 +38,13 @@ class BaseLoggedInView(View):
         locator='//div[@class="toast-notifications-list-pf"]')
     validations = ValidationErrors()
     current_user = Text("//a[@id='account_menu']")
+    ACCOUNT_MENU = Text("//a[@id='account_menu']")
+    LOGOUT = Text("//a[@href='/users/logout']")
```

Comment:
Is there a reason for these two to be in all-caps?

Reply:
copy paste :D